shims/cdpdc71/driver/pom.xml

```diff
@@ -183,6 +183,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
Since it looks like hive only uses this tool to generate its unit tests and not at runtime, delete this block.

Reply:
removed.